pygmt/clib/session.py

```diff
@@ -1375,7 +1375,7 @@ def virtualfile_from_data(
         check_kind : str
             Used to validate the type of data that can be passed in. Choose
             from 'raster', 'vector' or None. Default is None (no validation).
-        data : str, xarray.DataArray, 2d array, or None
+        data : str or xarray.DataArray or {table-like} or None
```

Comment:
`{table-like}` in this docstring is not rendered.

https://pygmt-git-standardize-table-like-docs-gmt.vercel.app/api/generated/pygmt.clib.Session.virtualfile_from_data.html

Reply:
Good catch. Maybe `{table-like}` needs to be placed last? I'll try with this:

```suggestion
        data : None or str or xarray.DataArray or {table-like}
```

Reply:
The replacement takes place in the fmt_docstring decorator. So I believe you have to add the `@fmt_docstring` decorator before the `virtualfile_from_data` function to make it work.

Reply:
You're right, forgot about that! I've added the `@fmt_docstring` decorator in 983dc16676d27f9fc04029191da0e5e01b98f54b so it should render properly now. Had to take away the docstring's curly brackets from the dictionary and f-strings though to prevent it from being formatted :joy: